cyberark-secrets-provider-for-kubernetes/templates/role.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: {{ include "conjur.rbac-api" . }}
+kind: Role
+metadata:
+  name: {{ .Values.rbac.roleName }}
+  namespace: {{ .Release.namespace }}
```

Comment:
Does the secrets provider only update secrets in the same namespace in which it runs?

Reply:
Yes indeed (: